mobile/src/views/HomeView.jsx

```diff
@@ -56,9 +57,10 @@ const HomeView = ({navigation}) => {
 
         <VendorList
           searchPhrase={searchPhrase}
-          data={vendorData}
+          data={vendorData.slice(0,3)}
```

Comment:
why do we need to slice the vendor data?

Reply:
To limit the amount of Vendor Cards appearing in HomePage